docs/src/manual/nonlinear.md

```diff
@@ -417,8 +417,12 @@ using the [`@operator`](@ref) macro.
 ## [User-defined operators](@id jump_user_defined_operators)
 
 In addition to a standard list of univariate and multivariate operators
-recognized by the `MOI.Nonlinear` submodule, JuMP supports *user-defined*
-Julia operators.
+recognized by the `MOI.Nonlinear` submodule, JuMP supports user-defined
+operators.
```

Comment:
At this point it's not clear what one can do with an operator that cannot be done with function tracing